src/app/common/components/presentation/check-box/check-box.component.ts

```diff
@@ -7,6 +7,8 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CheckBoxComponent implements OnInit {
 
+  label = 'Remember my credentials'
```

Comment:
This label should be dynamic and passed as @Input(). Before merging these all presentation components need to be completed or at least in a reusable state.